orchagent/main.cpp

```diff
@@ -249,8 +249,9 @@ int main(int argc, char **argv)
     /* Initialize orchestration components */
     DBConnector *appl_db = new DBConnector(APPL_DB, DBConnector::DEFAULT_UNIXSOCKET, 0);
     DBConnector *config_db = new DBConnector(CONFIG_DB, DBConnector::DEFAULT_UNIXSOCKET, 0);
```

Comment:
>new [](start = 29, length = 3)

There are memory leak risk at extreme memory situation.

Reply:
@qiluo-msft Hi Qi, could you elaborate the concern here?

Reply:
There are several 'new' statements here. If memory is used up in the middle, there will be memory leak issue.
The bug should be here for a long time, and your added code make it worse a little bit.

---
In reply to: [186248077](https://github.com/Azure/sonic-swss/pull/494#discussion_r186248077) [](ancestors = 186248077)